gtda/diagrams/features.py

```diff
@@ -391,3 +392,190 @@ def transform(self, X, y=None):
             Xt = np.linalg.norm(Xt, axis=1, ord=self.order).reshape(-1, 1)
 
         return Xt
+
+
+@adapt_fit_transform_docs
+class ComplexPolynomial(BaseEstimator, TransformerMixin):
+    """Coefficients of the complex polynomials whose roots are the points on the
+    persistence diagrams.
+
+    Given a persistence diagram consisting of birth-death-dimension triples
+    [b, d, q], subdiagrams corresponding to distinct homology dimensions are
+    considered separately, and their respective complex polynomial are computed
+    using the persistence points as roots. The :attr:`n_coefficients`
+    coefficients of those polynomial are returned as a vector of their real
+    parts concatenated with a vector of their imaginary parts [1]_.
+
+    **Important note**:
+
+        - Input collections of persistence diagrams for this transformer must
+          satisfy certain requirements, see e.g. :meth:`fit`.
+
+    Parameters
+    ----------
+    polynomial_type : ``'R'`` | ``'S'`` | ``'T'``, optional, default: \
+        ``'R'``
+        Type of complex polynomial to compute.
```

Comment:
This should be documented, i.e. what does R, S and T mean?

Reply:
Would the pointer to the paper be enough, since the file where they are defined is `_features`?

Reply:
I've just checked the paper and the transformations are simple enough to write down in the docstring in my opinion. I'll propose something later.

Reply:
So not in the description of parameters, but somewhere before `**Important note**`?

Reply:
Yes IMO, as it is an important piece of information to even understand the default